### DISCUSSION TITLE
Potato-Based Chat

### DISCUSSION THREAD

**Discussion by JasonWarrenUK** [FAC29A/hot-gitato#9] [General]
Let's use this to chat and keep the README as an archive-type doc

**Comment by Luciensday** [comment]
Help! I am confused what what is the current main is?

The pull request that I sent showing there Is a conflict.      It looks like it's compare Jason's html file ( as the spelling are corrected) 


<img width="933" alt="Screenshot 2023-09-15 at 22 58 30" src="https://github.com/FAC29A/hot-gitato/assets/128807685/cfe559b5-ac4d-41f2-a233-8762d18b98a5">



<img width="1223" alt="Screenshot 2023-09-15 at 22 59 15" src="https://github.com/FAC29A/hot-gitato/assets/128807685/5f5e5af9-4e39-4723-96c0-9e44496d454c">





I try to solve the conflict by pulling the latest main to my VS code. However what I see is the origin main that Frans created. below is the main that I see on vs code 



<img width="683" alt="Screenshot 2023-09-15 at 22 59 40" src="https://github.com/FAC29A/hot-gitato/assets/128807685/f03d421c-21eb-4374-becd-64aad75a43b0">



I can't solve the conflict because there is no conflict between my branch (add-styling)  and Frans' main branch.  On the other hand, GitHub pull request ask me to solve the conflict between my mine and Jason's spelling corrected main branch.  Can anyone try to pull the main to local and check what version of main branch is it? ( Frans' or Jason's)

**Reply by JasonWarrenUK** [reply]
Hey mate,

You merged my commit into `main`. I then deleted my branch. 

So the `main` here on GitHub is now the changes I made.

Before you pull, make sure you have done

`git checkout main`

And then checkout back to your branch once you've resolved that.

There will be a merge conflict because your branch includes code that I later changed.

I'm not certain why your local `main` branch still shows Fran's original version. 

Maybe @yuqingwwang or @nichgalzin can help?

**Reply by Luciensday** [reply]
I've done the `git checkout main`  but my local still shows Fran's original version of main branch. 

 Fran has solved the conflict of my branch on GitHub today.  My branch is now ready to merge ( I think it need your approval and act for merge) 

Once my branch is merge to the main - I will do  `git checkout main`  again and see if I got the latest version of main branch.
